tests/qa_automation/patch_and_reboot.pm

```diff
@@ -19,19 +19,26 @@ use warnings;
 use utils;
 use testapi;
 use qam;
+use power_action_utils 'power_action';
+use Utils::Backends 'use_ssh_serial_console';
 
 sub run {
     my $self = shift;
 
-    select_console 'root-console';
+    if (check_var('BACKEND', 'ipmi')) {
+        use_ssh_serial_console;
+    }
+    else {
+        select_console 'root-console';
+    }
 
     pkcon_quit unless check_var('DESKTOP', 'textmode');
 
     add_test_repositories;
 
     fully_patch_system;
 
-    type_string "reboot\n";
+    power_action('reboot', textmode => 1);
```

Comment:
are you sure it is in all cases textmode ?

Reply:
can it be something else when test is starting with root console ?

Reply:
I can just leave `type_string "reboot\n";` after `console('root-ssh')->kill_ssh;` on ipmi backend

Reply:
Used previous `type_string "reboot\n";` with `console('root-ssh')->kill_ssh;` on ipmi